app/facades/websocket/WebSocketController.scala

```diff
@@ -174,14 +199,37 @@ class WebSocketController @Inject() (actorSystem:  ActorSystem,
       Future.successful(Left[Result, DSAFlow](Forbidden))
   }
 
-  private def acceptOrResult(f: Option[DSLinkSessionInfo] => Future[Either[Result, Flow[DSAMessage, DSAMessage, _]]]): WebSocket = {
+  private def acceptOrResult(f: Option[DSLinkSessionInfo] => Future[Either[Result, DSAFlow]]): WebSocket = {
```

Comment:
in case you inject a custom type, please, don't forget to check all code places to be certain you changed it with the new type (currently it is DSAFlow), like 
test/models/rpc/DSAValueMsgpackSpec.scala:78

Reply:
@ilya-martynov , the type Flow[DSAMessage, DSAMessage, _] is used in test/models/rpc/DSAValueMsgpackSpec.scala only. In the tests i think it is more meaningful then just DSAFlow, so i don't think it has to be changed to DSAFlow.